WordPress/Sniffs/VIP/DirectDatabaseQuerySniff.php

```diff
@@ -167,7 +167,8 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
 		}
 
 		$whitelisted_cache = false;
-		$cached            = $wp_cache_get = false;
+		$cached            = false;
+		$wp_cache_get      = false;
```

Comment:
WordPress/Sniff.php, you have split out the multiple assignments as A = X, and then B = A.

Here, you have a pattern of A = X and B = X.

Is there any reason for this inconsistent approach? (There may well be, and that's fine if so)